build.go

```diff
@@ -119,3 +129,45 @@ func (b buildJob) List(filter M) (printer.Table, error) {
 func (b buildJob) Log(id string, writer io.Writer) error {
 	return b.clientImpl.logJob("build", id)
 }
+
+type buildReport struct {
+	Report string `json:"report"`
+}
+
+func (b buildJob) Report(id string) (string, error) {
+	var req = b.apiRequest(endpoints.builds.Report())
+	req.param("id", id)
+	resp, err := req.Do("GET", nil)
+	if err != nil {
+		return "", err
+	}
+	switch resp.StatusCode {
+	case 404:
+		return "", errors.New("Report not found")
+	case 204:
+		return "", errors.New("No report generated. Reports are only generated for COMPLETED builds")
+	case 200:
+		break
+	default:
+		return "", errors.New("Unknown error occured")
+	}
+
+	var apiResp struct {
+		Value struct {
+			Report string `json:"report"`
+		} `json:"value"`
+	}
+	err = json.NewDecoder(resp.Body).Decode(&apiResp)
+	if err != nil {
+		return "", err
+	}
+
+	validJSONReport := strings.Replace(apiResp.Value.Report, "\\", "", -1)
+	var prettyJSONReport bytes.Buffer
+	err = json.Indent(&prettyJSONReport, []byte(validJSONReport), "", "\t")
+	if err != nil {
+		return "", err
+	}
+
+	return string(prettyJSONReport.Bytes()), nil
```

Comment:
[`prettyJSONReport.String()`](https://golang.org/pkg/bytes/#Buffer.String).

Reply:
Or if we were on go1.10, [`strings.Builder`](https://golang.org/pkg/strings/#Builder) instead.